docs/source/transaction_family_specifications/integerkey_transaction_family.rst

```diff
@@ -0,0 +1,134 @@
+the value of entries stored in state. 
+
+An IntegerKey family transaction request is defined by the following values:
+
+* A verb which describes what action the transaction takes 
+* A name of the entry that is to be set or changed
+* A value by which the entry will be set or changed
+
+The 'set' verb is used to create new entries. The initial value of the entry will be 
+set to the value specified in the transaction request. The 'inc' and 'dec' verbs are 
+used to change the value of existing entries in state.
+
+State
+=====
+This section describes in detail how IntegerKey transaction information is stored 
+and addressed.
+
+The address values of IntegerKey transaction family entries are stored in state as
+a cbor encoded dictionary, with the key being the *Name* and the value being an integer 
+*Value*.
+
+.. code-block:: python
+
+    cbor({    
+        // The name of the entry : The current value of the entry
+        'Name':'Value'
+    })
+
+\ *Names* and *Values* must conform to the following rules:
+
+* Valid *Names* are utf-8 encoded strings with a maximum of 20 characters
+* Valid *Values* must be integers in the range of 0 through 2\ :sup:`32` − 1 (32-bit unsigned int)
+
+Addressing
+----------
+When an IntegerKey family transaction entry is stored, it can be accessed by addressing using 
+the following algorithm:
+
+The address of an IntegerKey family transaction entry must be a 70 character hexadecimal string. 
+Addresses for the IntegerKey transaction family are generated by adding the last 64 characters 
+of the sha512 hash of the entry *Name* to the first 6 characters of a sha512 hash of the 
+IntegerKey namespace (the IntegerKey namespace is the string "intkey"). For example, an 
+address could be generated like this:
+
+.. code-block:: pycon
+
+    >>> hashlib.sha512('intkey'.encode('utf-8')).hexdigest()[0:6] + hashlib.sha512('eBuPof'.encode('utf-8')).hexdigest()[:64]
+    '1cf126c584128aaf1837c90c83748ab222c11b8bbd2fe6cc30f17fe35f2acb9af8efd4'
+
+Transaction Payload
+===================
+
+IntegerKey transaction request payloads are defined by the following cbor data format:
+
+.. code-block:: python
+
+    cbor({
+        // Describes the action the transaction takes, either 'set', 'inc', or 'dec'
+        'Verb': 'verb',
+        
+        // The variable name of the entry which is to be modified
+        'Name': 'name',
+
+        // The amount to set, increment, or decrement
+        'Value': 1234,
+    })
+
+Transaction Header
+==================
+
+Inputs and Outputs
+------------------
+
+The inputs for IntegerKey family transactions must include:
+
+* Address of the *Name* being changed or added
+
+
+The outputs for IntegerKey family transactions must include:
+
+* Address of the *Name* being changed or added
+
+Dependencies
+------------
+* A list of transaction *header_signatures* which are required dependencies and must be processed prior to processing this transaction
```

Comment:
You might want to add "For example, 'inc' and 'dec' transactions might want to list the initial 'set' transaction for the *Name*. If 'inc' or 'dec' transactions are ordered before the corresponding 'set' transaction (and don't list the 'set' transaction as a dependency, they will be considered invalid (because *Name* will not exist when they are processed)."